sky/resources.py

```diff
@@ -1340,4 +1340,14 @@ def __setstate__(self, state):
                 state['_ports'] = resources_utils.simplify_ports(
                     [str(port) for port in original_ports])
 
+        if version < 14:
+            # Backward compatibility: we change the default value for TPU VM to
+            # True in version 14 (#1758), so we need to explicitly set it to
+            # False when loading the old handle.
+            if state.get('accelerators', '').startswith('tpu'):
```

Comment:
Can accelerators be a dict like
```
accelerators:
  tpu-v2-8: 1
```
?

Reply:
Ahh, good catch! Fixed and now tested it. : )